README.md

```diff
@@ -27,165 +27,142 @@ and signs AMP documents as requested by the AMP Cache.
+
+  1. Launch `amppkg` as a restricted user.
+  2. Save its stdout to a rotated log somewhere.
+
+Once you've done the above, you should be able to launch Chrome without any
+flags; just make sure chrome://flags/#enable-signed-http-exchange is enabled.
+
+#### Replicating
+
+If you need to set up multiple replicas of `amppkg` for scale, you'll want your
+`OCSPCache` to be backed by a shared storage device (e.g. NFS). It doesn't need
+to be shared among all instances globally, but perhaps among all instances per
+datacenter. The reason for this is to reduce the number of OCSP requests
+`amppkg` needs to make, per [OCSP stapling
+recommendations](https://gist.github.com/sleevi/5efe9ef98961ecfb4da8).
+
+#### How will these web packages be discovered by Google?
+
+For now, the presence of the `Vary: AMP-Cache-Transform` response header on an
+AMP HTML page will cause the Google AMP Cache to make a second request with
+`AMP-Cache-Transform: google` for the SXG.
+
+In the future, Googlebot may make all requests with `AMP-Cache-Transform: google`,
+eliminating the second fetch. It would be able to "unwrap" these SXGs and use
+them for Search.
 
 ### Limitations
 
 Currently, the packager will refuse to sign any AMP documents larger than 4 MB.
+Patches that allow for streamed signing are welcome.
 
-The Go http.Client implementation doesn't support redirect chains that rely on
-cookies being set and then read later in the chain; all requests are made
-cookielessly.
+If the packager refuses to sign any URL that results in a redirect. This is by
+design, as neither the original URL nor the final URL makes sense as the signed
+URL.
 
 To account for possible clock skew in user agents, the packager back-dates
 packages by 24h, which means they effectively last only 6 days for most users.
 
 This tool only packages AMP documents. To sign non-AMP documents, look at the
 commandline tools on which this was based, at
-https://github.com/WICG/webpackage/tree/master/go/signedexchange. This should be
-seen as a reference implementation, and not a supported library. Proper usage
-requires following the status of browser implementations and updating callers to
-import the correct library snapshot.
+https://github.com/WICG/webpackage/tree/master/go/signedexchange.
 
 ## Local Transformer
 
-The local transformer is a sub-library within the AMP Packager that transforms AMP HTML for security and performance improvements. These modifications are described in more detail [here](https://github.com/ampproject/amphtml/blob/master/spec/amp-cache-modifications.md).
+The local transformer is a sub-library within the AMP Packager that transforms AMP HTML for security and performance improvements. These modifications are described in more detail [here](https://github.com/ampproject/amphtml/blob/master/spec/amp-cache-modifications.md). Note that the transformed AMP HTML produced by the library is only valid inside of a signed exchange, and not to be served this as normal HTML.
```

Comment:
nit: I would drop "sub-" from "sub-" library. It doesn't clarify anything. All sub-libraries are also libraries, and vice-versa depending on usage.